packages/vaadin-overlay/test/position-target.test.js

```diff
@@ -109,6 +126,12 @@ describe('position target', () => {
       expectEdgesAligned(TOP, TOP);
     });
 
+    it('should align top edges when overlay part is animated', async () => {
```

Comment:
Added a test to make sure overlay part animations don't mess up the overlay positioning. The issue was originally discovered with select and fixed [here](https://github.com/vaadin/web-components/pull/2497/commits/262067b30b7d159ed90a6a9437f5858145e78735).